opencga-analysis/src/main/java/org/opencb/opencga/analysis/variant/VariantWalkerToolExecutor.java

```diff
@@ -0,0 +1,123 @@
+            toolParams.setOutputFileName(toolParams.getOutputFileName() + ".gz");
+        }
+
+        Container container = generateDockerObject();
+        checkDockerObject(container);
+    }
+
+    @Override
+    public void run() throws ToolException {
+        step(VariantWalkerToolExecutor.ID, () -> {
+            Path outDir = getOutDir();
+            String outputFile = outDir.resolve(toolParams.getOutputFileName()).toString();
+            Query query = toolParams.toQuery();
+            QueryOptions queryOptions = new QueryOptions()
+                    .append(QueryOptions.INCLUDE, toolParams.getInclude())
+                    .append(QueryOptions.EXCLUDE, toolParams.getExclude());
+            variantStorageManager.walkData(outputFile,
+                    format, query, queryOptions, dockerImage, toolParams.getCommandLine(), token);
+        });
+
+    }
+
+    private Container generateDockerObject() throws CatalogException, ToolException {
+        OpenCGAResult<ExternalTool> result;
+        if (externalToolParams.getVersion() != null) {
+            Query query = new Query(ExternalToolDBAdaptor.QueryParams.VERSION.key(), externalToolParams.getVersion());
+            result = catalogManager.getExternalToolManager().get(getStudy(), Collections.singletonList(externalToolParams.getId()), query,
+                    QueryOptions.empty(), false, token);
+        } else {
+            result = catalogManager.getExternalToolManager().get(getStudy(), externalToolParams.getId(), QueryOptions.empty(), token);
+        }
+        if (result.getNumResults() == 0) {
+            throw new ToolException("Variant walker tool '" + externalToolParams.getId() + "' not found");
+        }
+        ExternalTool externalTool = result.first();
+
+        if (externalTool == null) {
+            throw new ToolException("Variant walker tool '" + externalToolParams.getId() + "' is null");
+        }
+        if (externalTool.getType() != ExternalToolType.VARIANT_WALKER) {
+            throw new ToolException("User tool '" + externalToolParams.getId() + "' is not of type " + ExternalToolType.VARIANT_WALKER);
+        }
+        if (externalTool.getContainer() == null) {
+            throw new ToolException("User tool '" + externalToolParams.getId() + "' does not have a docker object");
+        }
+
+        return externalTool.getContainer();
+    }
+
+    private void checkDockerObject(Container container) throws ToolException, CatalogException {
+        if (org.apache.commons.lang3.StringUtils.isEmpty(container.getName())) {
+            throw new ToolException("Missing docker image name");
+        }
+        this.dockerImage = container.getName();
+        String tag = "";
+        if (org.apache.commons.lang3.StringUtils.isNotEmpty(container.getTag())) {
+            tag = container.getTag();
+            if (org.apache.commons.lang3.StringUtils.isNotEmpty(container.getDigest())) {
```

Comment:
Wait a minute.. why is this using an unqualified import.. ? 😱 
```
import org.apache.solr.common.StringUtils;
```

That's from the original (now "deprecated") variant-walker 🙃 
Should use the commons-lang version instead of a random one